docs/pages/_document.tsx

```diff
@@ -21,18 +20,9 @@ const DynamicLoadAnalytics = dynamic<any>(() =>
 export default class MyDocument extends Document<{ css?: string }> {
   static async getInitialProps(ctx: DocumentContext) {
```

Comment:
We can probably get rid of this

Reply:
Thank you for the hint, I will look at this and prepare next PR if needed!